.github/workflows/tests.yml

```
@@ -43,6 +43,7 @@ jobs:
           - "3.8"
           - "3.9"
           - "3.10"
+          - "3.11"
```

Or... Are you waiting for 3.11?
```suggestion
          - "3.11-dev"
```

---

Yes, I usually merge such PRs when the target version is available on GitHub Actions. I confirmed locally that 3.11 works. The risk of introducing a change that works on 3.10 but not 3.11 are vanishingly small.